css/css-grid/parsing/grid-area-computed.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>CSS Grid Layout Test: getComputedValue().gridArea</title>
+<link rel="help" href="https://drafts.csswg.org/css-grid-1/#propdef-grid-area">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/css/support/computed-testcommon.js"></script>
+</head>
+<body>
+<div id="target"></div>
+<script>
+// auto
+test_computed_value("grid-area", "auto / auto / auto / auto");
+test_computed_value("grid-row", "auto / auto");
+test_computed_value("grid-column-end", "auto");
+
+// <custom-ident>
+test_computed_value("grid-row", "-zπ", "-zπ / -zπ");
+test_computed_value("grid-row-start", "AZ");
+test_computed_value("grid-column-start", "-_π");
+test_computed_value("grid-row-end", "_9");
+
+
+// <integer> && <custom-ident>?
+test_computed_value("grid-area", "1 / 90 -a- / auto / auto");
+test_computed_value("grid-row", "2 az / auto");
+test_computed_value("grid-column", "9 / -19 zA");
+test_computed_value("grid-row-start", "-19");
+test_computed_value("grid-row-start", "9 -Z_");
+test_computed_value("grid-column-start", "-44 Z");
+test_computed_value("grid-row-end", "1 -πA");
+test_computed_value("grid-column-end", "5 π_");
+
+// span && [ <integer> || <custom-ident> ]
+test_computed_value("grid-area", "span 2 i / auto / auto / auto");
+test_computed_value("grid-row", "span 2 / auto");
+test_computed_value("grid-column", "span i / auto", "span 1 i / auto");
+test_computed_value("grid-column-start", "span 1 i");
+test_computed_value("grid-row-end", "span 2 i");
+test_computed_value("grid-column-end", "span 2");
+// grid-row-start  "span i" computes to "span 1 i" on Blink/WebKit,
+// but computes to "span i" on Edge/Firefox.
```

Comment:
I have raised https://github.com/w3c/csswg-drafts/issues/3448

The spec simply says
"If the `<integer>` is omitted, it defaults to 1."
"Computed value: specified keyword, identifier, and/or integer"

Reply:
We have interop in all other cases in this PR (apart from https://bugzilla.mozilla.org/show_bug.cgi?id=1339672 and https://bugzilla.mozilla.org/show_bug.cgi?id=1500590 )

Reply:
Again, the `Computed Value` line never has anything to do with serialization.

Per the general principle of "omit omittable things", tho, `span i` should indeed serialize as `span i`, since the `1` is implicit.

Reply:
Tests added.

Reply:
Note that browsers leave `"span 1 i"` unchanged both in the specified value and getComputedStyle.

Blink's computed style representation doesn't record if the 1 was implicit or explicit. (I don't know about Safari.) This would need to change for both `"span i"` and `"span 1 i"` to be returned unchanged by getComputedStyle.